dist/styles-error.css

```diff
@@ -0,0 +1,78 @@
+/* ----------------------------------------------------------
```

Comment:
Don't know what you think @lexoyo but I'm not a fan of committing generated file. Same goes for js

Reply:
JS generated files are a required step to be all-browsers-compatible via babel and to generate JS from JSX files, right ? For CSS, it depends on what you are doing in fact. Importing libraries without thinking about what you really needs is a bad practice. But if you really think about what you are doing, scss is providing many essentials features I'm used to work with. And finally, scss architecture with especially variables importation is something that I can not work without.

Reply:
You should build it from styles-error.scss and not commit the result

Reply:
I know you *need* them to run the app but they shouldn't be on git. Users must build then start CE2

Reply:
Ahhhhhhhhh, I didn't understand that on my first read... Ok, I agree with that so tell me.